src/module/Calibrator.cpp

```diff
@@ -120,7 +123,7 @@ bool Calibrator::setBrightness(Brightness brightness)
 
     int r, g, b;
     controller.getRawColor(r, g, b);
```

Comment:
これ、controllerのカラーセンサーのgetRawColorを呼び出さないと
白と黒の基準値が決まっていないから、`controller.getRawColor()`の表示がおかしくなっちゃいますよ。

```suggestion
     controller.colorSensor.getRawColor(r, g, b);
```